packages/components/tree/src/model/useDragNode.ts

```diff
@@ -164,7 +164,10 @@ export function useDragNodeHandler({ props, ctx, el$, dropIndicator$, store }) {
   const treeNodeDragEnd = (event: DragEvent) => {
     const { draggingNode, dropType, dropNode } = dragState.value
     event.preventDefault()
-    event.dataTransfer.dropEffect = 'move'
+
```

Comment:
Maybe we can add a link to the bug in the comments.
https://bugzilla.mozilla.org/show_bug.cgi?id=1911486